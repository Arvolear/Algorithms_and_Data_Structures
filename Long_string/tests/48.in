38870
36643956584906087109151597