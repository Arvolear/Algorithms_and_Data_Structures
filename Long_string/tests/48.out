36643956584906087109190467
36643956584906087109112727
1424350592455299605932722575390
942731067273117754287